packages/node_modules/@ciscospark/spark-core/src/lib/credentials/credentials.js

```diff
@@ -143,6 +159,15 @@ const Credentials = SparkPlugin.extend({
     Reflect.apply(SparkPlugin, this, args);
   },
 
+  downscope(scope) {
```

Comment:
This should probably be _underscore prefixed; `getToken(scope)` is how a consumer should cause a downsope

Reply:
no, don't do that. we're about the outlaw underscore prefixing :)